crates/catalog/rest/src/catalog.rs

```diff
@@ -1017,31 +1109,31 @@ mod tests {
             .with_summary(Summary {
                 operation: Operation::Append,
                 other: HashMap::from_iter([
-                                          ("spark.app.id", "local-1646787004168"),
-                                          ("added-data-files", "1"),
-                                          ("added-records", "1"),
-                                          ("added-files-size", "697"),
-                                          ("changed-partition-count", "1"),
-                                          ("total-records", "1"),
-                                          ("total-files-size", "697"),
-                                          ("total-data-files", "1"),
-                                          ("total-delete-files", "0"),
-                                          ("total-position-deletes", "0"),
-                                          ("total-equality-deletes", "0")
-                ].iter().map(|p|(p.0.to_string(), p.1.to_string())))
+                    ("spark.app.id", "local-1646787004168"),
```

Comment:
Nit: shouldn't this be caught by the linter? Same for the trailing comma's below.

Reply:
Sorry, I don't quite get your point.

Reply:
Why is this changed? It looks only indentation has changed, and most linters would not allow that since it makes the linter non-deterministic.

Reply:
Seems rust-fmt not enforcing the indentation.